ec2-ansible/inventory.ini

```diff
@@ -0,0 +1,2 @@
+[ec2]
```

Comment:
Lets use dynamic inventory

Reply:
I have updated the code to use dynamic inventory.